common.jsx

```diff
@@ -23,21 +49,36 @@ function widget(widgetName, widgetProps, key) {
 
 function href(widgetName, linkProps) {
   linkProps = { ...linkProps };
+
   if (props.nearDevGovGigsContractAccountId) {
     linkProps.nearDevGovGigsContractAccountId =
       props.nearDevGovGigsContractAccountId;
   }
+
   if (props.nearDevGovGigsWidgetsAccountId) {
     linkProps.nearDevGovGigsWidgetsAccountId =
       props.nearDevGovGigsWidgetsAccountId;
   }
+
   if (props.referral) {
     linkProps.referral = props.referral;
   }
+
   const linkPropsQuery = Object.entries(linkProps)
+    .filter(([_key, nullable]) => (nullable ?? null) !== null)
```

Comment:
I think it might be as simple as this:

```suggestion
    .filter(([_key, value]) => value != null)
```

Reply:
I'm checking for `undefined` values as well

Reply:
Plus, using `==` instead of `===` makes the logic to be quite unpredictable in JS, given the dynamic nature of its type system. The common practice is to always use `===` until you have a very clear reason to allow implicit type conversion.